### DISCUSSION TITLE
Can't connect to copilot from VSCode!

### DISCUSSION THREAD

**Discussion by amirrosenfeld** [community/community#125377] [Copilot Conversations]
### Select Topic Area

Bug

### Body

type: 'system',
  _name: 'FetchError',
  code: 'ETIMEDOUT',
  errno: -60,
  erroredSysCall: 'connect'

**Comment by gitstua** [comment]
👋🏽 Hi @amirrosenfeld 

If you are not using a corporate proxy (i.e. this is not at work) then it may be worth asking your ISP for assistance just in case it is something at their end.
